Dockerfile

```diff
@@ -0,0 +1,29 @@
+# start with ubuntu
+FROM python:3.11.7
+
+# install python3 and pip3
+RUN apt-get update && apt-get install -y python3 python3-pip python3-venv
```

Comment:
This can be removed entirely.

Reply:
bro was installing python even though while using the `python` image :skull:

Reply:
Done